README.md

```diff
@@ -25,22 +27,26 @@ Or install it yourself as:
 
 ## Dependencies
 
-Chasqui uses Redis to queue events and manage subscriptions. You can install
-Redis with your favorite package manager, such as homebrew, yum, or apt, or if
-you prefer, you can run `vagrant up` to run Redis in a virtual machine.
+Chasqui uses [Redis](http://redis.io/) to queue events and manage
+subscriptions. You can install Redis with your favorite package manager, such
+as homebrew, yum, or apt, or if you prefer, you can run `vagrant up` to run
+Redis in a virtual machine.
 
 ## Quick Start
 
-Chasqui consistents of two components - a client and a broker. The broker's
-responsibility is to forward published events to registered subscribers. The
-client can both publish events and register subscribers.
+Chasqui consists of two components - a client and a broker. Clients can both
+publish to a channel and subscribe to events on one or more channels. The
+broker transforms incoming events into Resque (or Sidekiq) jobs and places them
+on one or more queues according to the currently registered subscribers. Under
+the hood, a subscriber is simply a Sidekiq/Resque worker that processes jobs
+placed on a queue.
 
 ### Start the broker
 
     chasqui -r redis://localhost:6379/0 -q my-app
 
-Your broker must use the same Redis connection as your Sidekiq (or Resque)
-workers. For a list of available broker options, see `chasqui --help`.
+Your broker must use the same Redis connection as your Sidekiq/Resque workers.
```

Comment:
This is one of the things I got stuck on pretty early. Once I understood what was going on it made more sense, but yeah... But what do you mean by connection? I'm pretty sure you just mean must connect to the same database.

Thinking about it, I would actually describe this as Chasqui's killer feature. I think that this "*requirement*" is what would really get someones attention on why they would want to use Chasqui.

*Chasqui publishes directly to your worker queues!*